package/gluon-wlan-encryption-psk/Readme.md

```diff
@@ -0,0 +1,15 @@
+# Gluon wlan encrypt
+Encrypt wireless networks in a non-Freifunk setup.
```

Comment:
Enables encryption for AP and 802.11s mesh STA wireless networks using pre-shared keys defined in the site.conf at firmware build time.
This could also be used as description of the package - Don't mention Freifunk in this context.